### PR TITLE
Bump kubectl-fuzzy version to v1.9.0

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/d-kuro/kubectl-fuzzy/releases/tag/v1.9.0

Bump kubectl-fuzzy version to v1.9.0

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/fuzzy.yaml` is not a straightforward version bump: change on line `7` does not seem like a version bump: [`+    bin: kubectl-fuzzy`]

File: plugins/fuzzy.yaml [hunk 1]
```diff
@@ -16,8 +16,20 @@ spec:
...
+    bin: kubectl-fuzzy
+    files:
+    - from: kubectl-fuzzy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
```
Comment (d-kuro) [inline plugins/fuzzy.yaml:30]: Added darwin/arm64 build

**Pull request merged**